cli/src/cli_util.rs

```diff
@@ -2767,15 +2767,16 @@ impl LogContentFormat {
 pub fn get_new_config_file_path(
     config_source: ConfigSource,
     command: &CommandHelper,
-) -> Result<PathBuf, CommandError> {
+) -> Result<&Path, CommandError> {
```

Comment:
Since it returns a value borrowed from `command`, do you think that should become the first argument? Should the function even be moved onto `CommandHelper` or maybe onto `ConfigEnv`? I'm leaning towards moving it onto `ConfigEnv`.

Reply:
Yeah, moving to `ConfigEnv` sounds good. It can also be moved to `commands::config::ConfigLevelArgs`. iirc, there are some google-internal use of this function, but I assume they'll just need `new_<user|repo>_config_path()`, not a mapping from `ConfigSource` to file path?

(I'll send a separate PR, so marked as resolved.)

Reply:
Oh, good memory! I had completely forgotten that. We use it for writing a user-level config file based on their corp username and email.